### ISSUE TITLE
DockNode->ChildNodes missing

### ISSUE THREAD

**Issue by ptxmac** [AllenDang/cimgui-go#270]
See https://github.com/cimgui/cimgui/blob/docking_inter/generator/output/structs_and_enums.json#L6953
I assume this is because of general missing support for arrays?

**Comment by gucio321** [comment]
```console
[cimgui-go (0) ]$ make cimgui &> /tmp/cache.txt;cat /tmp/cache.txt |grep ChildNodes
2024-03-11 12:08:09	[DEBG]:	Unknown argument type "ImGuiDockNode*[2]" in function ImGuiDockNode_SetChildNodes
2024-03-11 12:08:09	[FAIL]:	(gengo_funcs.go:69):	not generated: ImGuiDockNode_SetChildNodes(ImGuiDockNode *ImGuiDockNodePtr, ImGuiDockNode* v)
2024-03-11 12:08:09	[DEBG]:	Unknown return type "ImGuiDockNode*[2]" in function ImGuiDockNode_GetChildNodes
```

You're right. I have to say that our generator still lacks many features e.g. this one.

**Comment by gucio321** [comment]
I've added something in https://github.com/AllenDang/cimgui-go/pull/272

**Comment by gucio321** [comment]
but it still needs return_type

**Comment by gucio321** [comment]
fixed in #274
